db-engine/src/main/java/com/splicemachine/db/iapi/types/UserType.java

```diff
@@ -698,7 +723,23 @@ public void read(Row row, int ordinal) throws StandardException {
 			setToNull();
 		else {
 			isNull = false;
-			value = row.get(ordinal);
+			byte[] bytes = (byte[]) row.get(ordinal);
+			ByteArrayInputStream bis = new ByteArrayInputStream(bytes);
+			ObjectInput in = null;
+			try {
+				in = new ObjectInputStream(bis);
+				value = in.readObject();
+			} catch (Exception e) {
+				throw new RuntimeException(e);
+			} finally {
+				try {
+					if (in != null) {
+						in.close();
+					}
+				} catch (IOException ex) {
+					// ignore close exception
+				}
+			}
```

Comment:
Here's another spot where a try-with-resources block can simplify things a bit.
```java
		else {
			isNull = false;
			byte[] bytes = (byte[]) row.get(ordinal);
			ByteArrayInputStream bis = new ByteArrayInputStream(bytes);

			try (ObjectInput in = new ObjectInputStream(bis)) {
				value = in.readObject();
			} catch (IOException | ClassNotFoundException e) {
				throw new RuntimeException(e);
			}
		}
```
Again, why are you throwing `RuntimeException` instead of `StandardException`?